stable/vmware-exporter/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- kremers
```

Comment:
You need at least two approvers/reviewers. (got the same comment on adding a new chart)

Reply:
Thx! Added sameersbn as suggested in the PR here